content/home.json

```diff
@@ -1,4 +1,30 @@
 {
+  "maintainers": {
+    "list": [
+      {
+        "name": "Bartłomiej Płotka",
+        "project": "Prometheus",
+        "src": "bartlomiej-plotka"
+      },
+      {
+        "name": "Chrissy LeMaire",
+        "project": "dbatools",
+        "src": "chrissy-lemaire"
+      },
+      {
+        "name": "Eileen Uchitelle",
+        "project": "Ruby on Rails",
+        "src": "eileen-uchitelle"
+      },
+      {
+        "name": "Helen Hou-Sandí",
+        "project": "WordPress",
+        "src": "helen-hou-sandi"
+      },
+      { "name": "Ifiok Jr.", "project": "Remirror", "src": "ifiok-jr" },
+      { "name": "Lorena Mesa", "project": "PyLadies", "src": "lorena-mesa" }
+    ]
+  },
```

Comment:
Maybe we could a "featured" attribute to the maintainers list in `maintainers.json` to avoid duplicating information, WDYT?

Reply:
That would be nice!

Reply:
That depends on the kind of the structure we want to load, if we're going for a `feature` attribute I'll recommend separating `maintainer` into a single file, we can have a meeting tomorrow or on Friday to talk about this!